miso-web/src/main/webapp/scripts/pool_ajax.js

```diff
@@ -830,4 +830,87 @@ Pool.barcode = {
       }
     );
   }
-};
\ No newline at end of file
+};
+
+Pool.orders = Pool.orders || {
+  'makeXhrRequest': function (method, endpoint, callback, data, callbackarg) {
+    var expectedStatus;
+    var unauthorizedStatus = 401;
+    if (method == 'POST') {
+      expectedStatus = [201, 200]; // will be 200 in case of a POST to sampleclass which triggers a GET for samplevalidrelationship
+    } else {
+      expectedStatus = [200, 404];
+    }
+    var xhr = new XMLHttpRequest();
+    xhr.open(method, endpoint);
+    xhr.onreadystatechange = function() {
+      if (xhr.readyState === XMLHttpRequest.DONE) {
+        if (expectedStatus.indexOf(xhr.status) != -1) {
+          if (!callback) {
+            document.location.reload();
+          } else {
+            data ? ( callbackarg ? callback(callbackarg) : callback() ) : callback(xhr) ;
+          }
+        } else if (xhr.status === unauthorizedStatus) {
+          alert("You are not authorized to view this page.");
+        } else {
+          alert("Sorry, something went wrong. Please try again. If the issue persists, contact your administrator.");
+        }
+      }
+    };
+    xhr.setRequestHeader('Content-Type', 'application/json');
+    data ? xhr.send(data) : xhr.send();
+  },
+  'addOrder': function(poolId) {
+    Pool.orders.makeXhrRequest('POST', '/miso/rest/poolorder', Pool.orders.loadOrders, JSON.stringify({ 'poolId': poolId, 'partitions': document.getElementById('newOrderParitions').value, 'parameters': { 'id': document.getElementById('newOrderParameterId').value } }), poolId);
+    return false;
+  },
+  'loadOrders': function(poolId) {
+    Pool.orders.makeXhrRequest('GET', '/miso/rest/pool/' + poolId + '/orders', function(xhr) {
+      var orders = JSON.parse(xhr.responseText);
+      document.getElementById('orderlist').innerHTML = orders.map(function(order) {
+        var platformOptions = Defaults.all.platforms.map(function(platform) {
+          return '<option value="' + platform.id + '"' + ((platform.id == order.parameters.platformId) ? " selected" : "") + '>' + platform.nameAndModel + '</option>';
```

Comment:
`<option>`s need to be wrapped in a `<select>` (right now `#orderlist` is a div)